tcf_website/static/css/site/pages/browse.css

```
@@ -105,19 +105,26 @@
 
+.school-header__heading {
+  margin: 0;
+}
+
 .school-header {
   display: flex;
   align-items: center;
   justify-content: space-between;
   gap: var(--space-4);
 
+  width: 100%;
```

Stylelint (17.14.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)